powerauth-webflow/package.json

```diff
@@ -19,43 +19,44 @@
   },
   "homepage": "https://github.com/wultra/powerauth-webflow",
   "dependencies": {
-    "axios": "^1.4.0",
+    "@snyk/protect": "^1.1080.0",
+    "axios": "^0.27.2",
+    "bluebird": "^3.7.2",
+    "intl": "^1.2.5",
+    "lodash": "^4.17.21",
     "react": "^16.14.0",
     "react-bootstrap": "^0.33.1",
-    "react-select": "1.3.0",
+    "react-device-detect": "^2.2.2",
     "react-dom": "^16.14.0",
+    "react-flags-select": "^2.2.3",
```

Comment:
Let's revert all changes except for `react-flags-select`, because many of the dependencies are a downgrade.

Reply:
package.json updated and currently contains no downgrades.